### ISSUE TITLE
[a11y][SideNavigationItem]: Tooltip not announced in Chrome JAWS

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11249]
### Bug Description

While using JAWS as a screen reader in Chrome, tooltips are not announced when interacting with elements.
In this case, the tooltip content is expected to be announced when the user focuses on the element. However, JAWS in Chrome fails to announce the tooltip. This creates an accessibility gap for users relying on JAWS in Chrome.

Link to an isolated example provided.

### Affected Component

SideNavigationItem

### Expected Behaviour

JAWS should announce the tooltip content consistently when the user focuses on the interactive element

### Isolated Example

https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cblxuPGhlYWQ-XG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cblxuICAgIGh0bWwge1xuICAgICAgZm9yY2VkLWNvbG9yLWFkanVzdDogbm9uZTtcbiAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1zaWRlLW5hdmlnYXRpb24gPiAgXG5cdFx0PHVpNS1zaWRlLW5hdmlnYXRpb24taXRlbSB0ZXh0PVwiSG9tZVwiaWNvbj1cImhvbWVcIiB0b29sdGlwPVwidG9vbHRpcCAxMjNcIj48L3VpNS1zaWRlLW5hdmlnYXRpb24taXRlbT5cblx0XHQ8dWk1LXNpZGUtbmF2aWdhdGlvbi1pdGVtIHNsb3Q9XCJmaXhlZEl0ZW1zXCIgdGV4dD1cIkhpc3RvcnlcIiBcbiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICBpY29uPVwiaGlzdG9yeVwiIHRvb2x0aXA9XCJ0b29sdGlwIDQ1NlwiPjwvdWk1LXNpZGUtbmF2aWdhdGlvbi1pdGVtPlxuICAgIDwvdWk1LXNpZGUtbmF2aWdhdGlvbj5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZCAtLT5cbiAgICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCJtYWluLmpzXCI-PC9zY3JpcHQ-XG48L2JvZHk-XG5cbjwvaHRtbD5cbjwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cbiAgIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb24uanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1NpZGVOYXZpZ2F0aW9uSXRlbS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb25Hcm91cC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2lkZU5hdmlnYXRpb25TdWJJdGVtLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9TaWRlTmF2aWdhdGlvbkdyb3VwLmpzXCI7XG5cbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2hvbWUuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2NoYWluLWxpbmsuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2dyb3VwLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9sb2NhdGUtbWUuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2NhbGVuZGFyLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9oaXN0b3J5LmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9jdXN0b21lci5qc1wiOyJ9LCJtYWluLmNzcyI6eyJuYW1lIjoibWFpbi5jc3MiLCJjb250ZW50IjoidWk1LXNpZGUtbmF2aWdhdGlvbiB7XG4gICAgaGVpZ2h0OiA2MDBweDtcbn0ifX0

### Steps to Reproduce

1. Create SideNavigation with SideNavigationItem
2. Add tooltip with any text to SideNavigationItem
3. Access SideNavigationItem with JAWS

### Priority

None

### UI5 Web Components Version

2.7.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

The tooltip of the side navigation item is not read. It's also scratched from the ACC tab in the chrome dev tools.

I tested the tooltip prop on a button component and its read properly by JAWS.

Could you please look into this issue ?

Regards,
Plamen Ivanov

**Comment by LidiyaGeorgieva** [comment]
Hello @AlekseiSch ,

We found out that this is a JAWS screen reader issue.
I isolated it with simple HTML and if you test with NVDA (for example) the tooltip is read out as expected, but with JAWS - not.
<div role="tree" aria-label="My tree">
	<div title="tooltip 123" tabindex="0" role="treeitem">Text</div>
</div>
Therefore I'm closing this issue.

Best Regards,
Lidiya
